server/service.go

```diff
@@ -256,6 +287,8 @@ func (m *BoostService) handleRegisterValidator(w http.ResponseWriter, req *http.
 		}(relay)
 	}
 
+	go func() { m.validatorRegistrationCh <- payload }()
```

Comment:
i wonder if we shouldn't do a buffered channel instead, perhaps with a select to skip if full 🤔

Reply:
i wonder if the channel is even needed, since it just kicks off a bunch of goroutines anyway.

this could be replaced with a function call, like 

```go
go m.sendValidatorRegistrationsToRelayMonitors(payload)
```

Reply:
you then move contention on the channel to contention for memory w/ a potentially unbounded number of goroutines...  I have made a note to ponder a similar issue with a use of channels in the relay-monitor... there is likely some common solution to handle the backpressure, it will just likely make the code look less nice ;)